test/Libraries/RevitTestServices/RevitNodeTestBase.cs

```diff
@@ -43,7 +44,8 @@ public override void Setup()
 
         protected override TestSessionConfiguration GetTestSessionConfiguration()
         {
-            return new TestSessionConfiguration(Dynamo.Applications.DynamoRevitApp.DynamoCorePath);
+            var asmLocation = AppDomain.CurrentDomain.BaseDirectory;
+            return new TestSessionConfiguration(Dynamo.Applications.DynamoRevitApp.DynamoCorePath, DynamoRevit.findRevitASMVersion(asmLocation));
```

Comment:
@QilongTang @aparajit-pratap this change is not decided on yet - please let me know what you think. This change makes it so all RevitNodeTests ignore the config file and always loads the host libG version (or should testing this takes a while as nugets must be generated).

I've also modified dynamo in my latest PR so that a simple alternative is to provide the config file but leave the version set to anything but a valid version - this results in the config scanning all installed asm products - on a test machine this will work in most cases.

The crux of the issue is between flexibility and robustness I think.